code/modules/projectiles/guns/ballistic/revolver.dm

```diff
@@ -296,8 +296,8 @@
 	user.visible_message(span_danger("[user.name]'s soul is captured by \the [src]!"), span_userdanger("You've lost the gamble! Your soul is forfeit!"))
 
 /obj/item/gun/ballistic/revolver/reverse //Fires directly at its user... unless the user is a clown, of course.
-	name = /obj/item/gun/ballistic/revolver/syndicate::name
-	desc = /obj/item/gun/ballistic/revolver/syndicate::desc
+	name = /obj/item/gun/ballistic/revolver/badass::name
+	desc = /obj/item/gun/ballistic/revolver/badass::desc
```

Comment:
Clown revolver should be the same as the default traitor revolver, not the nuke ops one
```suggestion
```

Reply:
Clown operatives use this which was my reasoning for keeping it as is, but it seems like Clown traitors get access to it as well. But I agree with making it look like a normal revolver regardless, so that even on Clown Ops it's difficult to differentiate the revolvers.